backend/src/data/repositories/user/user.repository.ts

```diff
@@ -15,6 +15,20 @@ class User {
     return this.#UserModel.query();
   }
 
+  async getByEmail(email: string): Promise<UserM | null> {
+    const user = this.#UserModel
+      .query()
+      .select()
+      .where({ email })
+      .first() as unknown as UserM | null;
```

Comment:
```suggestion
      .first() as UserM | null;
```

Reply:
![image](https://user-images.githubusercontent.com/78655088/183283082-7c58fe0b-31e9-494c-bfe3-b50b6a19939d.png)

typescript doesn't let me do this in such way

Reply:
okay, left it for now

Reply:
isn't resolved